packages/lib/constants.ts

```diff
@@ -68,5 +68,6 @@ export const IS_STRIPE_ENABLED = !!(
 );
 /** Self hosted shouldn't checkout when creating teams unless required */
 export const IS_TEAM_BILLING_ENABLED = IS_STRIPE_ENABLED && (!IS_SELF_HOSTED || HOSTED_CAL_FEATURES);
+export const IS_ORGANIZATION_BILLING_ENABLED = IS_STRIPE_ENABLED && (!IS_SELF_HOSTED || HOSTED_CAL_FEATURES);
```

Comment:
Not sure if we need this, maybe we can reuse TEAM_BILLING 🤔

Reply:
Team billing can be reused imho. 

Theyre the same conditional